projects/wave-core-new/src/lib/project/workspace-settings/workspace-settings.component.html

```diff
@@ -48,4 +48,6 @@ <h4 mat-line>New</h4>
         <h4 mat-line>Spatial Reference</h4>
         <p mat-line>Change the current spatial reference</p>
     </mat-list-item>
+    <h3 mat-subheader>Notifications</h3>
+    <wave-notifications></wave-notifications>
```

Comment:
please put a button here instead that leads to a new view with only the notifications shown. They can then use up the whole height of the page

Reply:
05d871119a540a3430c3ee6566796ed0a59d1fb6